### PR TITLE
Add Zabbix 3.0 support

### PR DESCRIPTION
**Pull request by oN0**
Hello!

Due to changes in Zabbix 3.0 API - added small patch.
Reason: removed ExpandData parameter in method trigger.get - https://support.zabbix.com/browse/ZBXNEXT-2724

Thanks.

### CODE REVIEW

File: Nagstamon/Nagstamon/Server/Zabbix.py [hunk 1]
```diff
@@ -261,6 +261,11 @@ def _get_status(self):
                     'triggerid': service['triggerid'],
                 }
 
+                if api_version > '3.0':
```
Comment (HenriWahl) [inline Nagstamon/Nagstamon/Server/Zabbix.py]: should it be ">" or ">="?

Comment (oN0) [inline Nagstamon/Nagstamon/Server/Zabbix.py]: Hmm, good question - I tried to pre-save the same code style as above in the code.

As they switched to 3-digit API version notation (3.0.0 returns now) both ">" and ">=" works, but if we will change version to "3.0.0" in code we should use only ">=".

Honestly, I'm not sure which one is more correct.

Comment (HenriWahl) [inline Nagstamon/Nagstamon/Server/Zabbix.py]: So ">=" seems more correct - also better understandable for readers of thid code later if version has to be at least 3.0, regardless of the upcoming minor versions.

Comment (oN0) [inline Nagstamon/Nagstamon/Server/Zabbix.py]: Yes, ">=" looks more reasonable.

Added.

**Comment by HenriWahl** [comment]
Can you port thisth the master tree too? Otherwise I will put your changes into https://github.com/HenriWahl/Nagstamon/blob/master/Nagstamon/Servers/Zabbix.py

**Comment by HenriWahl** [comment]
I added this to master tree. Can you check if the master versions runs in your setup too?

**Comment by oN0** [comment]
I've checked the master version - you also need to merge this commit [e9df689](https://github.com/oN0/Nagstamon/commit/e9df689a0ffd11d25a9e7075acc65f7d7d28c16c) to have it working.

**Comment by HenriWahl** [comment]
Applied, commited and pushed.

**Comment by waardd** [comment]
Hello,

Im testing nagstamon with zabbix 2.2 and 2.4

I found that the information events are shown as unknown....
Also if i check the filter for maintenance they still show up in the monitor. 

Still testing... I'm really glad with this tool! Keep up the good work.
If i find more i will let you know.

**Pull request merged**